src/wsrep_provider_v26.cpp

```diff
@@ -635,6 +637,23 @@ namespace
         // assert(not wsrep::tls_service_v1_probe(dlh));
         wsrep::tls_service_v1_deinit(dlh);
     }
+
+    static int init_ps_service(void* dlh,
+                               wsrep::ps_service* ps_service)
+    {
+        assert(ps_service);
+        if (not wsrep::ps_service_v1_probe(dlh))
```

Comment:
I have not seen this before, does this compile in our old compilers?

Reply:
fixed